docs/book/src/tasks/experimental-features/cluster-class/change-clusterclass.md

```diff
@@ -31,6 +29,7 @@ Once the changes are applied, the topology controller reacts as described in the
 | workers.machineDeployments[].infrastructure.ref | If the referenced template has changes only in metadata labels or annotations, the corresponding InfrastructureMachineTemplates are updated (in place update). <br /> <br />If the referenced template has changes in the spec:<br />  -  Corresponding InfrastructureMachineTemplate are rotated (create new, delete old).<br />  -  Corresponding MachineDeployments objects are updated with the reference to the newly created template (in place update). <br />  - The corresponding worker Machines are updated accordingly (rollout) |
 
 
-Note: In case a provider supports in place template mutations, the Cluster API topology controller will adapt to them at the next reconciliation, but the system is not watching for those specific changes. When the underlying template is updated in this way the changes may not be reflected immediately, but will be put in place at the next full reconciliation. The maximum time for the next reconciliation to take place is related to the CAPI controller sync period - 10 minutes by default. 
-
+Note: In case a provider supports in place template mutations, the Cluster API topology controller will adapt to them at the next reconciliation, but the system is not watching for those specific changes. When the underlying template is updated in this way the changes may not be reflected immediately, but will be put in place at the next full reconciliation. The maximum time for the next reconciliation to take place is related to the CAPI controller sync period - 10 minutes by default.
 
+<!-- links -->
+[Quick Start guide]: ../../user/quick-start.md
```

Comment:
I don't think this is needed here? The "Quick Start guide" link is actually in `docs/book/src/tasks/experimental-features/cluster-class/index.md`

Reply:
Thx, forgot to drop it when shuffling things around